### PR TITLE
Remove flakiness in VLMs 

### PR DESCRIPTION
**Pull request by zucchini-nlp**
# What does this PR do?

As per title. Only one test was flaky for VLMs - `test_promp_loopup_decoding`. Unfortunately it can't be fixed by suppressing image/video tokens because the Prompt Lookup doesn't apply any logits processor. This can be fixed with some additional effort, but not worth it. VLMs don't use these type of assisted decoding usually, haven't seen any issues from the community

### CODE REVIEW

File: tests/generation/test_utils.py [hunk 1]
```diff
@@ -1272,6 +1277,7 @@ def test_prompt_lookup_decoding_matches_greedy_search(self):
 
             config.is_decoder = True
             model = model_class(config).to(torch_device).eval()
+            logits_processors = self._get_logits_processor_kwargs(config=model.config)
```
Comment (ydshieh) [inline tests/generation/test_utils.py:1280]: so this is done to avoid some flakyness of a few non-VLM models, right?

File: tests/generation/test_utils.py [hunk 2]
```diff
@@ -1691,16 +1697,15 @@ def test_generate_from_inputs_embeds(self, _, num_beams):
...
+                model_name in model_class.__name__.lower() for model_name in VLM_CLASS_NAMES
             )
             if pixel_values_is_mutually_exclusive:
                 inputs_dict.pop("pixel_values", None)
                 inputs_dict.pop("pixel_values_videos", None)
                 inputs_dict.pop("pixel_values_images", None)
             #   2.C - No easy fix, let's skip the check that compares the outputs from `input_ids` and `inputs_embeds`
             has_complex_embeds_computation = any(
-                model_name in model_class.__name__.lower() for model_name in ["moshi", "qwen2vl", "qwen2_5_vl"]
+                model_name in model_class.__name__.lower() for model_name in ["moshi"]
```
Comment (ydshieh) [inline tests/generation/test_utils.py:1708]: I see
```
            if not has_complex_embeds_computation:
                self._check_similar_generate_outputs(outputs_from_ids, outputs_from_embeds)
```
a few lines below.

So "qwen2vl", "qwen2_5_vl" are actually now (in this PR) could run `_check_similar_generate_outputs`?

**Review by ydshieh — APPROVED** [review]
LGTM, thanks

File: tests/generation/test_utils.py (hunk 1 above)
Comment (zucchini-nlp) [reply tests/generation/test_utils.py:1280]: oops, will remove, this didn't help any model

**Comment by gante** [comment]
Thank you for fixing @zucchini-nlp 🤗
